firebase-perf/src/main/java/com/google/firebase/perf/metrics/AppStartTrace.java

```diff
@@ -173,15 +183,14 @@ static AppStartTrace getInstance(TransportManager transportManager, Clock clock)
     this.clock = clock;
     this.configResolver = configResolver;
     this.executorService = executorService;
-
-    StartupTime startupTime = FirebaseApp.getInstance().get(StartupTime.class);
-    if (startupTime == null) {
-      firebaseStartupTime = new Timer();
-    } else {
-      firebaseStartupTime =
-          Timer.ofElapsedRealtime(startupTime.getElapsedRealtime(), startupTime.getUptimeMillis());
-    }
     this.experimentTtid = TraceMetric.newBuilder().setName("_experiment_app_start_ttid");
+    this.processStartTime =
+        Build.VERSION.SDK_INT >= Build.VERSION_CODES.N
+            ? Timer.ofElapsedRealtime(Process.getStartElapsedRealtime())
+            : null;
+    StartupTime firebaseStart = FirebaseApp.getInstance().get(StartupTime.class);
+    this.firebaseClassLoadTime =
+        firebaseStart != null ? Timer.ofElapsedRealtime(firebaseStart.getElapsedRealtime()) : null;
```

Comment:
Can we leave some comments here on what this section is trying to do?

It might be helpful to qualify this section with a TODO that some of these can be cleaned up after the experiment analysis is done.

Reply:
Done